examples/llm/src/models/mosaic_gpt/mosaic_gpt.py

```diff
@@ -31,6 +32,8 @@
 from examples.llm.src.models.utils import (MODEL_INIT_REGISTRY,
                                            add_bidirectional_mask_if_missing)
 
+Tokenizer = Union[PreTrainedTokenizer, PreTrainedTokenizerFast]
```

Comment:
maybe import this type from common since it is also defined there?

Reply:
tbh this is also throwing lint errors:
```
error: Illegal type annotation: variable not allowed unless it is a type alias (reportGeneralTypeIssues)
```

I define Tokenizer as `Tokenizer: TypeAlias = Union[PreTrainedTokenizer, PreTrainedTokenizerFast]`

Reply:
couldn't get the linter to accept this...
sorry

Reply:
Lol all good